hack/verify-upstream-commits.sh

```diff
@@ -0,0 +1,46 @@
+#!/bin/bash
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+cd $(dirname "${BASH_SOURCE}")/..
+
+ok_revert_pat='UPSTREAM: [0-9]{4,}: revert origin [a-f0-9]{7,}:'
```

Comment:
I think we've been using `UPSTREAM: revert: <sha being reverted (remove this)>: <old pull number | carry | drop>`